### PR TITLE
Update docs to new standards and development environment to NTC stand…

### PR DESCRIPTION
**Pull request by itdependsnetworks**
…ards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,390 +1,66 @@
-# ntc-nautobot-plugin-nornir
+# Nautobot Plugin Nornir
 
-A plugin for [Nautobot](https://github.com/nautobot/nautobot), that intends to be a small shim layer between
-[nornir-nautobot](https://github.com/nautobot/nornir-nautobot) and other plugins. The primary abilities that the plugin provides is a native Nornir
-ORM based inventory and a credential manager.
+<p align="center">
+  <img src="./images/icon-NautobotPluginNornir.png" class="logo" height="200px">
```
Comment (cmsirbu) [inline README.md:4]:
```suggestion
  <img src="https://raw.githubusercontent.com/nautobot/nautobot-plugin-nornir/develop/docs/images/icon-NautobotPluginNornir.png" class="logo" height="200px">
```
I missed this in the cookie, image URL must be absolute so it also works in the docs site landing page.

Comment (itdependsnetworks) [reply README.md:4]: yea, a little bit of a catch 22, when rendering locally, this doesn't work until merged. I would prefer to keep a "TODO" in here.

Comment (itdependsnetworks) [reply README.md:6]: Understood, just doing it this way as I am testings.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -138,14 +186,14 @@ jobs:
       - name: "Upload binaries to release"
         uses: "svenstaro/upload-release-action@v2"
         with:
-          repo_token: "${{ secrets.GH_NAUTOBOT_BOT_TOKEN }}"
+          repo_token: "${{ secrets.NTC_GITHUB_TOKEN }}"
```
Comment (ubajze) [inline .github/workflows/ci.yml:189]: @itdependsnetworks I think we should keep the `GH_NAUTOBOT_BOT_TOKEN`

**Review by ubajze — CHANGES_REQUESTED** [review]

File: docs/user/app_feature_credentials.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  description: Group Definitions for device type SPINE
+  is_active: true
+  device-roles:
+    - slug: spine
+nautobot_plugin_nornir:
+  pluginx: 
+    connection_secret_path: "pluginx.extras.secret"
+```
+
+TODO: Verify `pluginx`
```
Comment (itdependsnetworks) [inline docs/user/app_feature_credentials.md:123]: @jeffkala

Comment (jeffkala) [reply docs/user/app_feature_credentials.md:123]: What is the ask here? `pluginx` could be any other nornir plugin.  for example `"pyntc.extras.secret"`

**Review by jeffkala — APPROVED** [review]

**Pull request merged**